sympy/core/basic.py

```diff
@@ -760,24 +807,25 @@ def _sorted_args(self):
         The same as ``args``.  Derived classes which don't fix an
         order on their arguments should override this method to
         produce the sorted representation.
+
         """
         return self.args
 
 
     def as_poly(self, *gens, **args):
-        """Converts ``self`` to a polynomial or returns ``None``.
-
-           >>> from sympy import sin
-           >>> from sympy.abc import x, y
+        """
+        Converts ``self`` to a polynomial or returns ``None``.
 
-           >>> print((x**2 + x*y).as_poly())
-           Poly(x**2 + x*y, x, y, domain='ZZ')
+        >>> from sympy import sin
+        >>> from sympy.abc import x, y
+        >>> print((x**2 + x*y).as_poly())
+        Poly(x**2 + x*y, x, y, domain='ZZ')
 
-           >>> print((x**2 + x*y).as_poly(x, y))
-           Poly(x**2 + x*y, x, y, domain='ZZ')
+        >>> print((x**2 + x*y).as_poly(x, y))
```

Comment:
I think that the changes like these can still be useful.
Can you open up a new PR containing only the changes about the bad indentation like these?

Reply:
Yes I will